bin/hdfcoinc/pycbc_plot_psd_file

```diff
@@ -68,8 +72,6 @@ for psd_file in args.psd_files:
                 raise ValueError("low_frequency_cutoff not found in hdf file; "
```

Comment:
maybe you would want to rewrite this error message as it's now not quite accurate - eg `"hdf file %s does not contain an attribute giving low frequency cutoff" % psd_file' .

Also `ValueError` is not quite the right error class, the docs describe it as "Raised when an operation or function receives an argument that has the right type but an inappropriate value".  Use `RuntimeError` for a generic error encountered during execution.

Reply:
thanks for the comments @tdent, I've made those changes.